src/frontend/magnify/packages/components/i18n/frontend.json

```diff
@@ -215,6 +215,18 @@
     "description": "Title for the room configuration section with general settings",
     "message": "Settings"
   },
+  "components.rooms.instantRoom.startButtonLabel": {
+    "description": "Label for the button to start an instant room",
+    "message": "Start"
+  },
+  "components.rooms.instantRoom.startInputPlaceHolder": {
+    "description": "Label for the input to start an instant room",
```

Comment:
Why are those translations edit in the turborepo commit ?

Reply:
I ran the command for translations to see if everything worked as before. Since I had to change the path